src/tests/ftest/util/dmg_utils.py

```diff
@@ -101,3 +102,104 @@ def run(self, timeout=None, verbose=True, env=None, sudo=False):
         """
         return process.run(self.__str__(), timeout, verbose, env=env,
                            shell=True, sudo=sudo)
+
+
+def clean_daos_mnt(hosts):
+    """Clean up daos mnts on servers.
+
+    Args:
+        hosts (list): list of host names where servers are running
+
+    Returns:
+    False if issue running command. True otherwise.
+
+    """
+    cleanup_cmds = [
+        "sudo umount /mnt/daos; sudo rm -rf /mnt/daos",
+        "rm -rf /tmp/daos_sockets/",
+        "rm -rf /tmp/*.log",
+    ]
+    # Intentionally ignoring the exit status of the command
+    result = pcmd(hosts, "; ".join(cleanup_cmds), False, None, None)
+    if len(result) > 1 or 0 not in result:
+        print("Error removing daos mount")
+        return False
+    return True
+
+def storage_scan(hosts, path=""):
+    """ Execute scan command through dmg tool to servers provided.
+
+    Args:
+        hosts (list): list of servers to run scan on.
+        path (str, optional): Path to dmg command binary. Defaults to "".
+
+    Returns:
+        False if issue running command. True otherwise.
+
+    """
+    # Create and setup the command
+    dmg = DmgCommand(path)
+    dmg.request.value = "storage"
+    dmg.action.value = "scan"
+    dmg.insecure.value = True
+    dmg.hostlist.value = hosts
+
+    try:
+        dmg.run()
+    except process.CmdError as details:
+        print("<dmg> command failed: {}".format(details))
+        return False
+
+    return True
```

Comment:
We do need to return the scan information for example NVMe drive mappings and use those mapping in the server file. so only returning True/False is not enough.
But for us the priority is to get the NVMe format first so if you want you can fix it with this PR or open new JIRA task and handle later.

Reply:
You are right. I made a change to this so that it returns the command object so we are able to get the stdout from the command execution.